src/constants.ts

```diff
@@ -8,4 +8,5 @@ export enum ROUTES {
   PROFILES_EXPLORER_VIEW = '/profiles-explorer',
   ADHOC_VIEW = '/ad-hoc',
   SETTINGS = '/settings',
+  METRICS_VIEW = '/metrics',
```

Comment:
```suggestion
  METRICS = '/metrics',
```

Don't forget to rebase to see the latest routes changes

Reply:
👍 done in https://github.com/grafana/explore-profiles/pull/360/commits/6b7e931f767e599fecfbd5f8183c4f2085f8dc15